roles/openshift_cluster_monitoring_operator/tasks/install.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Ensure that Monitoring Operator has nodes to run on
```

Comment:
It's probably good to be precise here and say the "Cluster Monitoring Operator"

Reply:
Done